apps/asap-server/src/utils/sync-google-calendar.ts

```diff
@@ -40,6 +38,11 @@ const fetchEvents = async (
     singleEvents: true, // recurring events come returned as single events
   };
 
+  if (!syncToken) {
+    params.timeMin = new Date('2020-10-01').toISOString();
```

Comment:
why don't you use DateTime for both dates?

Reply:
The native one is a pattern we are used to see and know.

Reply:
nah, im not convinced, i stand by what i said, mixing two different date constructors literally in two lines of code does not sound like a good practice, especially since we know at this point that using `Date` constructor with `new` is not testable.

Reply:
IT's a fixed date. That date does not change